README.md

```diff
@@ -30,6 +30,20 @@ Compute Engine:
        $ gcloud config set project <project>
     gcloud config set project ...
 
+    # If you would like to use service account instead please look here:
+    docker run --volume /.config/gcloud/ \
```

Comment:
alternatively you could just name the `gcloud auth activate-service-account` container instead of using a data container.

I don't mind either way, but we should be consistent with the rest of the README file, i.e: if we switch to data container (maybe with tianon/true, instead of busybox), we should also change the example below.